### DISCUSSION TITLE
[help] chia plots passed filter 1/8 why?

### DISCUSSION THREAD

**Discussion by mobinparsian** [Chia-Network/chia-blockchain#3163] [General]
hi .. im setup chia and fullnode and sync wallet and all of the page perview synced in the plots passed filter just show 1/8 a few times.. and in proof found 0 . why my plots passed filter not 8/8 ? 
and how can im undrestand how much i get cxh ? always show 0 .. im install chia about 2 week

**Comment by dorkmo** [comment]
the filter is a quick way to reduce the number of plots that are eligible to win. its normal to see 0/8 or 1/8. it would be very rare to see all of your plots pass the filter.

**Reply by mobinparsian** [reply]
hi thanks for answer ... i think have problem in port 8444 ... im send some photos please check and help me
![1](https://user-images.githubusercontent.com/83422774/116737070-037d4880-aa06-11eb-95a5-82a5f77b7301.png)
![2](https://user-images.githubusercontent.com/83422774/116737098-0aa45680-aa06-11eb-970a-2fe821e571db.png)
![3](https://user-images.githubusercontent.com/83422774/116737133-13952800-aa06-11eb-9477-27a0309c9245.png)
![4](https://user-images.githubusercontent.com/83422774/116737136-14c65500-aa06-11eb-959a-9ae2a8b0cc86.png)
![5](https://user-images.githubusercontent.com/83422774/116737144-1859dc00-aa06-11eb-9f19-cbba844e9abf.png)

**Reply by dorkmo** [reply]
looks like youre successfully connected using upnp. That last image is showing the connection on your local computer. You could open 8444, but it looks okay at the moment

**Comment by sargonas** [comment]
Tagging this issue for migration to a discussion. Apologies in advance, but you will be best served by moving this over there and out of the issue tracker, where the community can better assist you. (Ideally the issue tracker should be used to focus on bugs and development problems that need to be addressed, with general user support handled in Discussions and Keybase)

^MigrateToDiscussion
